packages/cron/src/lib/email/providers/mailchimp.js

```diff
@@ -24,7 +25,8 @@ export default class MailchimpEmailProvider {
 
   /**
      * Sends the email of the given type to the given recipient.
-     * Returns the unique message ID if successful, or undefined if the sending failed.
+     * Returns the unique message ID.
+     * Throws EmailSendError if the sending failed.
```

Comment:
Is this function now only returning Promise<string|undefined>? (line 36)

Reply:
Done